### ISSUE TITLE
EventBase::loop(): Failed to invoke event callback

### ISSUE THREAD

**Issue by 2ik** [walkor/workerman#703]
hello everyone! 
Today the project broke down, the reason was the overflow of disk space with logs:
```
Warning: EventBase::loop(): Failed to invoke event callback in ../vendor/workerman/workerman/Events/Event.php on line 195
```

who knows why this might happen?

My configs: 

```PHP
// index.php
$worker::$eventLoopClass = '\Workerman\Events\Event';
```

```
root@server:~# php -v
PHP 8.1.0 (cli) (built: Nov 25 2021 20:22:22) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.0, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.0, Copyright (c), by Zend Technologies

root@server:~# php -m
[PHP Modules]
calendar
Core
ctype
date
dom
event
exif
FFI
fileinfo
filter
ftp
gettext
hash
iconv
json
libxml
mbstring
openssl
pcntl
pcre
PDO
Phar
posix
readline
redis
Reflection
session
shmop
SimpleXML
sockets
sodium
SPL
standard
sysvmsg
sysvsem
sysvshm
tokenizer
xml
xmlreader
xmlwriter
xsl
Zend OPcache
zlib

[Zend Modules]
Zend OPcache
```

```ini
; php.ini
....
.......
opcache.enable=1
opcache.enable_cli=1
opcache.save_comments=0
opcache.enable_file_override=1
opcache.validate_timestamps=0
opcache.jit_buffer_size=512M
opcache.jit=1255

```

Thanks!

**Comment by 2ik** [comment]
I think this is a bad compatibility of the event library with php 8.
installed the beta version
`pecl install event-3.0.7RC1`

**Comment by walkor** [comment]
Please run command `php start.php status` and show the output.

**Comment by walkor** [comment]
I noticed event v3.0.6 with php8 has this problem, and fixed in 3.0.7RC1 see  https://bitbucket.org/osmanov/pecl-event/issues/69/php-8-compatibility .
